MedicalDocManagement.BLL/Services/ChildCardsService.cs

```diff
@@ -139,19 +143,38 @@ public ParentChildCardDTO AddParentIntoChildCard(ParentChildCardDTO parentChildC
 
             return parentChildCardDTO;
         }
-            
-        public List<ChildCardDTO> FindChildCards(ChildCardDTO childCardDTO)
+
+        public List<ChildCardDTO> FindFirst20ChildCards(IViewPatientData viewPatientData)
```

Comment:
Because I decided that for now it will be enough to output first twenty patients.